src/components/action/features/FeatureAction.js

```diff
@@ -0,0 +1,21 @@
+import React, { Component } from 'react';
+import Action from '../Action';
+import FeaturesSettings from './FeaturesSetings';
+
+class FeaturesAction extends Component {
+    render() {
+        console.log(this.props);
+        return (
+            <Action
+                title="Extract Features"
+                collapsible
+                checked={this.props.checked}
+                onToggleActive={this.props.onToggleActive}
+            >
+                <FeaturesSettings {...this.props} />
```

Comment:
Same as in Alex'es branch, let's introduce dedicated container for FeatureSettings. It's is easier to maintain code when mapStateToProps corresponds to propTypes of a react component (easier to notice changes, cleanup unnecessary props and as a result - unnecessary re-renderings)